airswap/airswap-web
# Fix/523 check wrapped swaps

Closes #523 

`Swap.check` wasn't being called for wrapped swaps because of a legacy issue that meant wrapped swaps weren't supported.

I've removed the condition and we're now calling `Swap.check` passing the wrapper contract as the `senderAddress` when necessary.

**Testing instructions**

- Verify that currently on `develop`, performing a swap for 1 ETH -> DAI will fail with the unpredictable gas limit error. This is because the maker server doesn't have enough DAI to service the order it proposes.
- Pull this branch and verify that the swap still fails but you are correctly shown the `SIGNER_BALANCE_LOW` error.
- Swapping with a smaller amount (e.g. 0.001 ETH) should work correctly.

**Bonuses!**

- Added prettier to devDependencies for version consistency between developers, and re-formatted all files with this version.
- Fixed some bad validator messages where the key didn't map to the correct description.
